contentcuration/contentcuration/models.py

```diff
@@ -1778,6 +1778,16 @@ def mark_complete(self):  # noqa C901
         self.complete = not errors
         return errors
 
+    def make_content_id_unique(self):
+        """
+        If self is NOT an original contentnode (in other words, a copied contentnode)
+        and a contentnode with same content_id exists then we update self's content_id.
+        """
+        is_node_original = self.original_source_node_id is None or self.original_source_node_id == self.node_id
+        does_same_content_exists = ContentNode.objects.exclude(pk=self.pk).filter(content_id=self.content_id).exists()
```

Comment:
I think this will be eagerly evaluated, so we could put this inline in the `and` statement, so that we don't have to do the query at all if `is_node_original` is `True`, because the `and` evaluation will shortcut.

Reply:
@rtibbles that's a good point sir! I'have pushed a commit to address this.